python/lsst/ts/standardscripts/base_take_image.py

```diff
@@ -189,6 +202,12 @@ async def configure(self, config):
                 nimages = 1
             self.config.exp_times = [self.config.exp_times] * nimages
 
+        if hasattr(config, "ignore"):
+            self.log.debug("Ignoring TCS components.")
+            self.tcs.disable_checks_for_components(components=config.ignore)
+            self.log.debug("Ignoring Camera components.")
+            self.camera.disable_checks_for_components(components=config.ignore)
```

Comment:
We should only ignore tcs components, please remove the feature to ignore camera components.